_data/tagging-status.yml

```diff
@@ -4375,12 +4375,14 @@
 
  - name: hypbmsec
    type: package
-   status: compatible
+   status: partially-compatible
    included-in: [arxiv001]
    priority: 7
    issues:
+   comments: Currently compatible, but once the new tagging interface for headings is in place this might stop working
```

Comment:
tagging interface -> template interface?

Reply:
well it is both , but yes, definitely better